app_book/forms.py

```diff
@@ -16,12 +16,23 @@ class BookForm(forms.ModelForm):
     widgets: Defintion wie die Eingabefelder auszusehen haben. Hier am Beispiel css Klasse von Bootstrap genutzt und Vorschau eingebaut
     labels: Definition was bei den Label Tags auf der Oberflaeche erscheinen soll. Wenn dies nicht definiert worde ist wird der Attributenname der Modellklasse genommen
     '''
+    delete_saved_image = forms.BooleanField(required=False, label='Gespeichertes Bild löschen')
```

Comment:
Kannst du diesen Haken in die Nähe zum Bild positionieren? Am besten ein "X" in die obere rechte Ecke vom Bild. Das wäre am schönsten.